packages/terra-tabs/src/Tabs.scss

```diff
@@ -0,0 +1,105 @@
+@import './ModularTabs';
+@import './StructuralTabs';
+
+:local {
+  .tabs-container {
+    width: 100%;
+  }
+
+  .collapsible-tabs-container,
+  .collapsed-tabs-container {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    position: relative;
+    width: 100%;
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .collapsible-tabs-container.is-calculating {
+    visibility: hidden;
+
+    /*
+     * During calculations tabs should display with auto size and not grow or shrink in order to get
+     * content width for trunction logic
+     */
+    .tab {
+      flex: 0 0 auto;
+    }
+  }
+
+  .tab,
+  .tab-menu {
+    cursor: pointer;
+    font-size: var(--terra-tabs-font-size, 1rem);
+    line-height: var(--terra-tabs-line-height, 1.4);
+    padding: var(--terra-tabs-padding, 0.643rem 1.071rem);
+  }
+
+  .tab,
+  .tab-menu span {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    word-break: normal;
+    word-wrap: normal;
+  }
+
+  .tab {
+    flex: 0 1 auto;
+    min-width: 8.357rem;
+    text-align: center;
+
+    &.is-icon-only {
+      flex: 0 0 auto;
+      min-width: 2.571rem;
+      padding: var(--terra-tabs-icon-only-padding, 0.5rem 0.857rem);
+    }
+
+    &.is-text-only {
+      min-width: 7.143rem;
+      padding: var(--terra-tabs-text-only-padding, 0.643rem 1.071rem);
+    }
+
+    .label:nth-child(2) {
+      margin-left: var(--terra-tabs-label-margin-next-to-icon, 0.357rem);
+    }
+  }
+
+  .tab-menu {
+    align-items: baseline;
+    display: flex;
+    flex: 1 1 auto;
+    min-width: 7.143rem;
+
```

Comment:
Need to add `justify-content: center;` to horizontally center the text for when "Menu" displays.